src/Mod/PartDesign/Gui/ViewProviderTransformed.cpp

```diff
@@ -158,11 +170,171 @@ bool ViewProviderTransformed::onDelete(const std::vector<std::string> &s)
     return ViewProvider::onDelete(s);
 }
 
+class ComputationDialog : public QDialog, public Message_ProgressIndicator {

```

Comment:
Would a [QProgressDialog](https://doc.qt.io/qt-6/qprogressdialog.html#details) be more appropriate?